Automattic/simplenote-macos
# StatusBar: Optionally Hidden

### Details:
In this PR we're implementing a new macOS MenuItem, capable of Displaying / Hiding the new StatusBar.

Closes #986
cc @eshurakov Sir!! almost there!! may I bug you with this one?

Thanks in advance!!

### Test: Toggling!
1. Launch Simplenote
2. Click over `View > Hide Status Bar`

- [x] Verify the Status Bar gets hidden
- [x] Verify that both the Editor and Notes List look great
- [x] Verify that clicking `Show Status Bar` reverts this change
- [x] Verify that if you relaunch the app, the last setting is preserved

### Test: Regressions
1. Launch Simplenote
2. Click over `View > Hide Sidebar`

- [x] Verify the Sidebar gets hidden
- [x] verify that repeating the process allows you to **Show Sidebar**

### Release
`RELEASE-NOTES.txt` was updated in 61b7ef5 with:

> The new Status Bar can also be hidden via Settings #988
